wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenItem.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { memo } from 'react';
```

Comment:
Is there anything new in this file or mostly split/copy from `AssetPicker/index.tsx`?

Reply:
Most of the file is split from `AssetPicker/TokenList.tsx`.

Added/Changed part is added token display name and explorer link.
As per design I'll have the USD value as well.